editor/src/components/canvas/commands/insert-element-insertion-subject.ts

```diff
@@ -39,31 +41,35 @@ export const runInsertElementInsertionSubject: CommandFunction<InsertElementInse
   let selectedViews: Array<ElementPath> = []
   const { subject, insertionPath } = command
 
+  let subjectElement: JSXElement = subject.element
```

Comment:
yes, I was worried that working with `subjectElement` for the editorPatch, but using `subject.element.uid` in the description might cause an inconsistency issue in the future (if, for example, for some reason it will be changed inside this function and then the description will have the wrong `uid` of the original one that's not actually in the patch)